datatrails-common-api/assets/v2/assets/eventresponse.proto

```diff
@@ -164,4 +167,17 @@ message EventResponse {
         "Identity of the tenant the that created this event"
         max_length: 1024
     }];
+
+    // An event has exactly one proof mechanism. This field caputures the proof
```

Comment:
Minor: I think comments like this are transposed onto the swagger when it gets autogenerated. We need verbose commentary like this, but not exposed to the user via swagger

Reply:
ah ok, is there a way to add the comment so it _doesn't_ get into the swagger ?

Reply:
My memory is hazy, but i think all comments directly above properties,messages or rpc definitions all go to swagger, so maybe a blank line might work?

Reply:
Will defer this for now but remember it. LMK what the best way is to capture the more verbose comentary